utils/unittest/execution_state.go

```diff
@@ -36,10 +36,11 @@ var ServiceAccountPublicKey flow.AccountPublicKey
 
 func init() {
 	var err error
-	GenesisStateCommitment, err = hex.DecodeString(GenesisStateCommitmentHex)
+	GenesisStateCommitmentBytes, err := hex.DecodeString(GenesisStateCommitmentHex)
```

Comment:
lets include a size check here ... just to be cautious of future code modifications.